package.json

```diff
@@ -69,6 +69,7 @@
     "react-google-charts": "^3.0.15",
     "react-map-gl": "^5.2.3",
     "react-mapbox-gl": "^4.8.2",
+    "react-router-dom": "^5.1.2",
```

Comment:
may remove it if not needed, Although there're a lot other redundant, but let's not add more